spotbugs/build.gradle

```diff
@@ -96,7 +96,7 @@ dependencies {
   api "com.google.code.gson:gson:2.8.9"
   testImplementation 'junit:junit:4.13.1'
   testImplementation 'org.apache.ant:ant:1.10.12'
-  testImplementation 'org.apache.logging.log4j:log4j-slf4j18-impl:2.15.0'
+  testImplementation 'org.apache.logging.log4j:log4j-slf4j18-impl:2.16.0'
```

Comment:
We have at least two places where log4j dependency is hard coded, we should extract it to a variable like `asmVersion` above

Reply:
Nice catch, I'll do it :)